src/sentry/userfeedback/endpoints/project_userfeedback_details.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api import serializers
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.serializers import serialize
+from sentry.models.project import Project
+from sentry.userfeedback.models import UserFeedback
+from sentry.userfeedback.serializers import UserFeedbackSerializer
+
+
+@region_silo_endpoint
+class ProjectUserFeedbackDetailsEndpoint(ProjectEndpoint):
+    publish_status = {
+        "DELETE": ApiPublishStatus.EXPERIMENTAL,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "PUT": ApiPublishStatus.EXPERIMENTAL,
+    }
+
+    def get(self, request: Request, project: Project, userfeedback_id: str) -> Response:
```

Comment:
Carrying this feedback over from the other pull.  Let's standardize on the `user_feedback` naming convention.